src/dune_rules/coq_sources.ml

```diff
@@ -14,6 +14,17 @@ type t =
   ; extract : Coq_module.t Loc.Map.t
   }
 
+let find_module ~source t =
+  let exception Found of Coq_lib_name.t * Coq_module.t in
+  try
+    let check_found name m =
+      if Path.Build.equal source (Coq_module.source m) then
+        raise (Found (name, m))
```

Comment:
Should be `raise_notrace`.

By the way, since this is a general purpose function, I'd move it to stdune's map module.

Reply:
OK, done.